packages/tools/pie-icons-webc/package.json

```diff
@@ -34,7 +43,7 @@
     "PIE"
   ],
   "scripts": {
-    "build": "rollup -c",
+    "build": "vite build && tsc",
```

Comment:
the `vite-dts` plugin was causing a lot of strange and unwanted behaviour with duplicate types, lacking maps and difficult to configure output locations.

Typescript can be configured to simply output the maps and d.ts files and it appears to be a lot simpler than trying to leverage vite for creating types.

I am unsure what benefit vite gives us for declaring types, so I would be tempted to suggest we use this approach for generating types across our components too.

Reply:
I assume that's why we were importing from the dist folder too right?
https://github.com/justeattakeaway/pie/pull/667#discussion_r1281555911

Reply:
discussed offline but it appears TS typings are working as expected 👍